islands/GlobalSearch.tsx

```diff
@@ -234,14 +234,10 @@ export default function GlobalSearch() {
                 {kinds.map((k) => (
                   <button
                     class={tw`px-2 rounded-md leading-relaxed hover:(bg-gray-100 text-main) ${
-                      // TODO: use border instead
                       k === kind
-                        ? css({
-                          "text-decoration-line": "underline",
-                          "text-underline-offset": "6px",
-                          "text-decoration-thickness": "2px",
-                        })
-                        : ""} ${k === kind ? "text-black" : "text-gray-500"}`}
+                        ? "border-black border-b-2 rounded-none text-black"
```

Comment:
it should still be rounded. it should look like this:
<img width="473" alt="Screenshot 2022-08-17 at 23 18 01" src="https://user-images.githubusercontent.com/13135287/185244974-e1f1fe18-3ff9-47df-9f2f-e15af9030127.png">